rdtools/degradation.py

```diff
@@ -10,62 +10,59 @@
 import statsmodels.api as sm
 
 
-def degradation_with_ols(normalized_energy):
+def degradation_ols(normalized_energy):
```

Comment:
Ah, should we then also add `degradation_year_on_year` and `degradation_classical_decomposition` as well?